index.d.ts

```diff
@@ -1,9 +1,12 @@
 declare module 'coordinate-parser' {
-    class Coordinates {
-        constructor(coordinateString: string)
-        getLatitude(): number
-        getLongitude(): number
-    }
+  class CoordinateParser {
+    constructor(coordinateString: string);
+    getLatitude(): number;
+    getLongitude(): number;
+  }
 
-    export default Coordinates
+  module CoordinateParser {}
```

Comment:
I think this line does nothing.

Reply:
Needs to be there. Otherwise you get `Module ''coordinate-parser'' resolves to a non-module entity [...]`

Reply:
Good to know.